.github/workflows/ci-cd-renv.yml

```diff
@@ -68,7 +76,7 @@ jobs:
 
       - name: Deploy to shinyapps.io
         # Continuous deployment only for pushes to the main / master branch
-        if: false && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/master')
+        if: startsWith(github.ref, 'refs/heads/') || github.ref == 'refs/heads/main' || github.ref == 'refs/heads/master'
```

Comment:
I will remove condition `startsWith(github.ref, 'refs/heads/')` before merging (also in the other workflow)

Reply:
done